packages/apollo-datasource-rest/src/types/http-cache-semantics/index.d.ts

```diff
@@ -1,4 +1,4 @@
-declare module 'http-cache-semantics' {
+declare namespace CachePolicy {
```

Comment:
The changes here would generally be cleaner to just switch to using the DefinitelyTyped definitions for this.  However, DefinitelyTyped is implementing the documented API and the existing code is unfortunately using undocumented/internal methods.